src/js/App/Header/UserToggle.js

```diff
@@ -11,7 +11,7 @@ import PropTypes from 'prop-types';
 import { connect } from 'react-redux';
 
 function buildItems(username, isOrgAdmin, accountNumber = -1, extraItems) {
-    return [
+    const dropdownItems = [
```

Comment:
I would prefer spreading instead of splicing
```JSX
return [
        <DropdownItem key="Username" isDisabled>
            <dl className='ins-c-dropdown-item__stack'>
                <dt className="ins-c-dropdown-item__stack--header">Username:</dt>
                <dd className="ins-c-dropdown-item__stack--value">{username}</dd>
            </dl>
        </DropdownItem>,
        <React.Fragment key="account wrapper">
            { accountNumber > -1 &&
                <DropdownItem key="Account" isDisabled>
                    <dl className='ins-c-dropdown-item__stack'>
                        <dt className="ins-c-dropdown-item__stack--header">Account Number:</dt>
                        <dd className="ins-c-dropdown-item__stack--value">{accountNumber}</dd>
                    </dl>
                </DropdownItem>
            }
        </React.Fragment>,
        ...isOrgAdmin ? [
          <DropdownItem
            key="User management"
            href={`https://www.${window.insights.chrome.isProd ? '' : 'qa.' }redhat.com/wapps/ugc/protected/usermgt/userList.html`}
            target="_blank" rel='noopener noreferrer'>
                User management
          </DropdownItem>
        ] : []
        <DropdownSeparator key="separator" />,
        <DropdownItem
            key="My Profile"
            href={`https://access.${window.insights.chrome.isProd ? '' : 'qa.'}redhat.com/user`}
            target="_blank"
            rel='noopener noreferrer'>
                My profile
        </DropdownItem>,
        <DropdownItem
            key="logout"
            component="button"
            onClick={() => window.insights.chrome.auth.logout(true)}>
                Logout
        </DropdownItem>,
        [...extraItems]
    ]
```